data/project.ttl

```diff
@@ -13,10 +13,10 @@
     schema:creator <https://edmo.seadatanet.org/report/422> .
 
 <https://orcid.org/0000-0003-0663-5907> a schema:Person ;
-    schema:name "Laurian van Maldghem"^^xsd:string .
+    schema:name "Laurian van Maldeghem"^^xsd:string .
 
 <https://orcid.org/0000-0002-9648-6484> a schema:Person ;
-    schema:name "Marc Portier"^^xsd:string .
+    schema:name "Marque Portier"^^xsd:string .
 
 <https://edmo.seadatanet.org/report/422> a schema:Organization ;
     schema:name "VLIZ vzw"^^xsd:string .
```

Comment:
actually something we might want to retrieve via deref?
quick test:
```
$ curl --url "https://edmo.seadatanet.org/report/422" -H "Accept: text/turtle" 
```
shows that LOD representation does exist, but no turtle is on offer (no conneg support), only html with embeded jsonld in script tag

think we best take 2 routes for this
1. ask seadatanet in current ldes project to also cover the conneg
2. check if we can have the dref thingy find triples in the html --> note that rdflib might do this out of the box (i.e. you might not need to scan script tags yourself)

Reply:
This issue will be tranported over to the derefernce branch once it exists